javascript/ql/src/semmle/javascript/dataflow/internal/FlowSteps.qll

```diff
@@ -529,3 +529,19 @@ module PathSummary {
    */
   PathSummary return() { exists(FlowLabel lbl | result = MkPathSummary(true, false, lbl, lbl)) }
 }
+
+/**
+ * Provides predicates for reasoning about calls to identity functions.
+ */
+module IdentityCalls {
+  /**
+   * Gets an identity call for `input` that can be recognized syntactically.
+   */
+  DataFlow::CallNode syntactic(DataFlow::Node input) {
```

Comment:
The naming is a little weird. From the call site it's not really clear what predicate does.

How about `FlowSteps::identityFunctionStep(pred, succ)` instead?